docker.mk

```diff
@@ -6,6 +6,7 @@
 # Common make targets for samples' Docker images.
 
 SAMPLE_REGISTRY        ?= docker.io/dapriosamples
+GHRC_REGISTRY          ?= ghrc.io/dapr/samples
```

Comment:
We can revert the changes in this file and simply set SAMPLE_REGISTRY=ghrc.io/dapr/samples in the workflow and reuse the existing make targets.